README.md

```diff
@@ -43,36 +97,47 @@ This ensures Playwright:
   - Runs the shared `setup` project (auth), then: 
   - Finally runs your deployment tests.
 
-2. Use the utilities in your tests
+## Write The Tests 
 
-```js
-import { dismissBetaBanner, getLtiIFrame } from '@oxctl/deployment-test-utils/testUtils'
+Use the utilities from this repository when writing your deployment tests. 
```

Comment:
Use the utilities from this package in your deployment tests. For example, to assert that some text is visible after the tool loads:

```js
import { test, expect } from '@playwright/test'
import { dismissBetaBanner, getLtiIFrame, waitForNoSpinners } from '@oxctl/deployment-test-utils'
```
etc. etc.